### DISCUSSION TITLE
💡 Flux.1-dev-Controlnet-Upscaler

### DISCUSSION THREAD

**Discussion by ZeroCool22** [comfyanonymous/ComfyUI#5101] [General]
https://huggingface.co/jasperai/Flux.1-dev-Controlnet-Upscaler

This is Flux.1-dev ControlNet for low resolution images developed by Jasper research team.

![examples_showcase](https://github.com/user-attachments/assets/d5ef379b-5d90-4158-ae15-63b48f479bbf)

**Comment by JorgeR81** [comment]
I haven't tried the Upscaler ControlNet yet, but their Depth ControlNet seems to be much better than the one from Shakker Labs.

https://huggingface.co/jasperai/Flux.1-dev-Controlnet-Depth

https://huggingface.co/Shakker-Labs/FLUX.1-dev-ControlNet-Depth
